README.md

```diff
@@ -11,6 +11,15 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 ### 3.0.0 (alpha)
 
 * Removed cloning from `MultiTerm.getTexts`.
+* Turned EntityId into a simple wrapper around an opaque serialization string
+  * Removed EntityId::getNumericId
+  * Removed EntityId::getEntityType
+  * Removed EntityId::getPrefixedId
+  * Introduced EntityId::getSerialization
```

Comment:
I find the use of `::` in JavaScript very confusing. When I write `EntityId.getNumericId` I'm describing a specific method in a specific class and do not say if it's static or not.